src/Symfony/Bundle/FrameworkBundle/Resources/config/templating.xml

```diff
@@ -58,5 +59,15 @@
         </service>
 
         <service id="templating.loader" alias="templating.loader.filesystem" />
+
+        <!--
+            This should be in templating_php.xml but unfortunately, Twig depends on this helper.
+            As the Twig extension depending on this service is deprecated, this will be removed in 3.0.
```

Comment:
Twig does not depend on it anymore, so the comment should be updated to reflect why we are adding this service here back.